src/lib/ui/thermostat/ClimatePresets.tsx

```diff
@@ -0,0 +1,222 @@
+        <Button
+          onClick={() => {
+            setSelectedClimatePreset(CreateNewPresetSymbol)
+          }}
+          className='seam-climate-presets-add-button'
+        >
+          <AddIcon />
+          {t.createNew}
+        </Button>
+
+        <div className='seam-thermostat-climate-presets-cards'>
+          {device.properties.available_climate_presets.map((preset) => (
+            <PresetCard
+              onClickEdit={() => {
+                setSelectedClimatePreset(preset)
+              }}
+              onClickDelete={() => {
+                setInDeletionPresetKey(preset.climate_preset_key)
+                deleteMutation.mutate({
+                  climate_preset_key: preset.climate_preset_key,
+                  device_id: device.device_id,
+                })
+              }}
+              temperatureUnit={props.temperatureUnit}
+              preset={preset}
+              key={preset.climate_preset_key}
+              deletionLoading={
+                deleteMutation.isPending &&
+                inDeletionPresetKey === preset.climate_preset_key
+              }
+              disabled={
+                deleteMutation.isPending &&
+                inDeletionPresetKey !== preset.climate_preset_key
+              }
+            />
+          ))}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+function PresetCard(
+  props: HTMLAttributes<HTMLDivElement> & {
+    preset: ThermostatClimatePreset
+    temperatureUnit: 'fahrenheit' | 'celsius'
+    onClickEdit: () => void
+    onClickDelete: () => void
+    deletionLoading?: boolean
+    disabled?: boolean
+  }
+): JSX.Element {
+  const {
+    preset,
+    temperatureUnit,
+    onClickEdit,
+    onClickDelete,
+    deletionLoading = false,
+    disabled = false,
+    ...attrs
+  } = props
+
+  const heatPoint =
+    temperatureUnit === 'fahrenheit'
+      ? preset.heating_set_point_fahrenheit
+      : (preset.heating_set_point_celsius ?? undefined)
+
+  const coolPoint =
+    temperatureUnit === 'fahrenheit'
+      ? preset.cooling_set_point_fahrenheit
+      : (preset.cooling_set_point_celsius ?? undefined)
+
+  const unitSymbol = getTemperatureUnitSymbol(temperatureUnit)
+
+  const chips = (
+    [
+      heatPoint != null
+        ? { icon: <ThermostatHeatIcon />, value: `${heatPoint} ${unitSymbol}` }
+        : undefined,
+      coolPoint != null
+        ? { icon: <ThermostatCoolIcon />, value: `${coolPoint} ${unitSymbol}` }
+        : undefined,
+      preset.fan_mode_setting != null
+        ? { icon: <FanIcon />, value: preset.fan_mode_setting }
+        : undefined,
+    ].filter(Boolean) as Array<{ icon: ReactNode; value: string }>
+  ).map(({ icon, value }, index) => (
+    <div key={index} className='seam-thermostat-climate-preset-chip'>
+      <span className='seam-thermostat-climate-preset-chip-icon'>{icon}</span>
+      <span className='seam-thermostat-climate-preset-chip-value'>{value}</span>
+    </div>
```

Comment:
This should just be a `Chip` component, and then you don't need this `.filter(Boolean) as Array<{ icon: ReactNode; value: string }` stuff, you can just do


```
Chip ({ icon, children }) {
   ...
}

// later

{heatPoint && <Chip icon={<ThermostatHeatIcon />}>`${heatPoint} ${unitSymbol}`></Chip>}